Symfony/CS/Tests/Fixer/Contrib/ModernizeTypesCastingFixerTest.php

```diff
@@ -0,0 +1,137 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Tests\Fixer\Contrib;
+
+use Symfony\CS\Tests\Fixer\AbstractFixerTestBase;
+
+/**
+ * @author Vladimir Reznichenko <kalessil@gmail.com>
+ */
+class ModernizeTypesCastingFixerTest extends AbstractFixerTestBase
+{
+    /**
+     * @dataProvider provideExamples
+     */
+    public function testFix($expected, $input = null)
+    {
+        $this->makeTest($expected, $input);
+    }
+
+    public function provideExamples()
+    {
+        $multiLinePatternToFix = <<<'FIX'
+<?php $x =
+intval
+
+(
+    mt_rand
+    (
+        0, 100
+    )
+
+)
+
+;
+FIX;
+        $multiLinePatternFixed = <<<'FIXED'
+<?php $x =
+(int) (
+    mt_rand
+    (
+        0, 100
+    )
+
+)
+
+;
+FIXED;
+
+        $overriddenFunction = <<<'OVERRIDDEN'
+<?php
+
+class overridesIntval
+{
+    public function intval($x)
+    {
+        return \intval($x);
+    }
+
+    public function usesInval()
+    {
+        return intval(mt_rand(0, 100));
+    }
+}
+OVERRIDDEN;
+
+        $overriddenFunctionFixed = <<<'OVERRIDDEN'
+<?php
+
+class overridesIntval
+{
+    public function intval($x)
+    {
+        return (int) $x;
+    }
+
+    public function usesInval()
+    {
+        return intval(mt_rand(0, 100));
```

Comment:
It does not work that way.
http://3v4l.org/6THPc

Here, `intval` will call global function, and `$this->intval` would call class' method.